client/apps/shipping-label/components/label-reprint-modal/index.js

```diff
@@ -59,6 +59,7 @@ ReprintDialog.propTypes = {
 	reprintDialog: PropTypes.object,
 	paperSize: PropTypes.string.isRequired,
 	storeOptions: PropTypes.object.isRequired,
+	labelId: PropTypes.number,
```

Comment:
Is this meant to be a part of these changes?

Reply:
Whoops — thanks!